### PR TITLE
[stable/apm-server] bump apm-server, fix docs

### PR DESCRIPTION
**Pull request by kimxogus**
Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/apm-server/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 description: The server receives data from the Elastic APM agents and stores the data into a datastore like Elasticsearch
 icon: https://www.elastic.co/assets/blt47799dcdcf08438d/logo-elastic-beats-lt.svg
 name: apm-server
-version: 0.1.0
-appVersion: 6.2.4
+version: 0.1.1
```
Comment (cpanato) [inline stable/apm-server/Chart.yaml]: lets bump to 1.0.0 since this chart is in the stable folder
```suggestion
version: 1.0.0
```

Comment (kimxogus) [reply stable/apm-server/Chart.yaml]: bumped to 1.0.0 :)

**Review by cpanato — APPROVED** [review]

**Pull request merged**